app/bootstrap.php

```diff
@@ -63,14 +64,15 @@ function (ConsoleCommandEvent $event) use ($userInteraction) {
 };
 
 $container['installer.docker'] = function ($c) {
+    $dinghyCli = new DinghyCli($c['process.interactive_runner']);
```

Comment:
I guess it doesn't matter, it can just be a different service if it's needed with another process runner